src/lighteval/utils.py

```diff
@@ -189,3 +189,14 @@ def is_peft_available() -> bool:
 
 
 NO_PEFT_ERROR_MSG = "You are trying to use adapter weights models, for which you need `peft`, which is not available in your environment. Please install it using pip."
+
+
+def can_load_extended_tasks() -> bool:
+    imports = []
+    for package in ["langdetect"]:
```

Comment:
So we would need to manually add needed packages in this list every time we add a new extended task ?

Reply:
Yes, I don't think there's an easy way to import the list from the pytoml. Maybe @lewtun has an idea?

Reply:
Yeah I don't think there is a simple way to check at runtime the whole list of extras. The way we handle this in `transformers` is to have helper methods like `_is_package_available()` ([link](https://github.com/huggingface/transformers/blob/ff841900e45763114d2417fb24ce29d950c6c956/src/transformers/utils/import_utils.py#L41)) which are then used in methods like `is_sentencepiece_available()` ([link](https://github.com/huggingface/transformers/blob/ff841900e45763114d2417fb24ce29d950c6c956/src/transformers/utils/import_utils.py#L765-L766))

I would recommend adopting a similar logic

Reply:
This is what we do for all the other packages, so I think we'll have to stick with adding the deps manually here for the extended tasks specifically

Reply:
Thanks a lot :)